src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -43,13 +43,30 @@
                 this.createEntryForm(component, response);
                 component.find('forceRecordCmp').reloadRecord(true);
             } else {
-                this.showToast(component, $A.get('$Label.c.PageMessagesError'), response.getReturnValue(), 'error');
+                this.handleApexErrors(component, response.getError());
+                this.hideFormSpinner(component);
             }
             this.hideSpinner(component);
         });
         $A.enqueueAction(action);
     },
 
+    /**
+     * @description: handles the display of errors from an apex callout
+     * @param errors: list of potential errors passed back from apex
+     */
+    handleApexErrors: function(component, errors) {
+        let message;
+        if (errors) {
```

Comment:
Any reason not to consolidate this with the next line? `if (errors && errors[0] && errors[0].message)`?